bin/git-undo

```diff
@@ -1,6 +1,28 @@
 #!/usr/bin/env bash
 
 back="^"
+cstr="commits"
+ccnt=$(git rev-list --count HEAD)
+if [ $ccnt -eq 1 ]; then cstr="commit"; fi
+parm3=""
+
+function _undo()
+{
+  type=${1:-soft}
+  undo_cnt=${2:-1}
+  reset=${3:-""}
+echo "type=$type, cnt=$undo_cnt, reset=$reset"
+  if [ $undo_cnt -gt $ccnt ]; then
+    echo "Only $ccnt $cstr, cannot undo $undo_cnt"
+  elif [ "$type" = "hard" -a $ccnt -eq $undo_cnt ]; then
```

Comment:
Better to use `[ "$type" = "hard" ] && [ $ccnt -eq $undo_cnt ]`, which is more clear.

Reply:
Well, I disagree that it's clearer, but it's not a hill worth dying on, so I'll change it.